pkg/metaserver/util/util.go

```diff
@@ -75,6 +75,7 @@ func UnsafeKindToResource(k string) string {
 	}
 	unusualKindToResource := map[string]string{
 		"Endpoints": "endpoints",
+		"Gateway":   "gateways",
```

Comment:
Where's the type `Gateway` from? If it's a type of crd, I think we can't list all types here.